app/enterprise/2.2.x/developer-portal/administration/application-registration/enable-key-auth-plugin.md

```diff
@@ -0,0 +1,145 @@
+
+| Form Parameter | Description                                                                       |
+|:---------------|:----------------------------------------------------------------------------------|
+| `Service` | The Service that this plugin configuration will target. Required. |
+| `Anonymous` | An optional string (Consumer UUID) value to use as an anonymous Consumer if authentication fails. If empty (default), the request fails with an `4xx`. Note that this value must refer to the Consumer `id` attribute that is internal to Kong, and **not** its `custom_id`. |
+| `Hide Credentials` | Whether to show or hide the credential from the Upstream service. If `true`, the plugin strips the credential from the request (i.e., the header, query string, or request body containing the key) before proxying it. Default: `false`. |
+| `Key in Body` | If enabled, the plugin reads the request body (if said request has one and its MIME type is supported) and tries to find the key in it. Supported MIME types: `application/www-form-urlencoded`, `application/json`, and `multipart/form-data`. Default: `false`. |
+| `Key Names` | Describes an array of parameter names where the plugin will look for a key. The client must send the authentication key in one of those key names, and the plugin will try to read the credential from a header, request body, or query string parameter with the same name. The key names may only contain [a-z], [A-Z], [0-9], [_] underscore, and [-] hyphen. Required. Default: `apikey`. |
+| `Run on Preflight` | Indicates whether the plugin should run (and try to authenticate) on `OPTIONS` preflight requests. Default: `true`. |
+
+## Generate a Credential {#gen-client-id-cred}
+
+Generate a Client ID credential to use as an API key. You can generate multiple
+credentials.
+
+1. In the **Dev Portal > My Apps** page, click **View** for an application.
+
+2. In the **Authentication** pane, click **Generate Credential**.
+
+   ![Application Authentication Pane](/assets/images/docs/dev-portal/gen-client-id-secret.png)
+
+   Now you can make requests using the Client ID as an API Key.
+
+## Make Requests with an API Key (Client Identifier)
+
+The Client ID of your credentials can be used as an API key to make authenticated requests to a Service.
+
+**Tip:** You can also access key request instructions directly within the user interface from the
+information icon in the Services details area of your application. Click the **i** icon to open the Service Details page.
+
+![Services Pane](/assets/images/docs/dev-portal/portal-info-modal-key-auth.png)
+
+Scroll to view all of the available examples.
+
+![Service Details Page Embedded Key Usage Instructions](/assets/images/docs/dev-portal/service-details-key-auth-usage.png)
+
+### Make a request with the key as a query string parameter
+
+{% navtabs %}
+{% navtab Using cURL %}
+
+```bash
+$ curl -X POST {proxy}/{route}?apikey={CLIENT_ID}
+```
+
```

Comment:
Remove blank lines around code example (83 and 86). Same goes for all subsequent other code examples in the {% navtabs codeblock %} style. 

(Can't add a suggestion, github gets confused by too many backticks in a row)

Reply:
> Remove blank lines around code example (83 and 86). Same goes for all subsequent other code examples in the {% navtabs codeblock %} style.
> 
> (Can't add a suggestion, github gets confused by too many backticks in a row)

Done in commit 4eb4be6